docs/pages/admin-guides/migrate-plans.mdx

```diff
@@ -219,6 +228,19 @@ To migrate Teleport agents:
 
 1. Stop Teleport services on each Agent (if applicable).
 
+1. If you are using Teleport repositories to install on Linux
+   update the repository channel to `stable/cloud` for Teleport
+   Enterprise (cloud). For self-hosted you can use the major version
+   `stable/v(=teleport.major_version=)` or if you are using the automatic
+   upgrades the `stable/rolling` has all released versions.
+
+1. If you are using the community version of Teleport uninstall `teleport`

```

Comment:
I would link to the instructions for uninstalling Teleport here

Reply:
> I would link to the instructions for uninstalling Teleport here

thanks, updated